app/assets/stylesheets/application.css.scss

```diff
@@ -28,3 +28,18 @@
 .downvote {
   color: red;
 }
+
+#foto, #foto img {
+    background: #1ABC9C;
```

Comment:
Color literals like `#1ABC9C` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#1ABC9C` should be written as `#1abc9c`<br>Line should be indented 2 spaces, but was indented 4 spaces